src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -43,27 +45,37 @@ private string FindProjectReference(XDocument document, string projectUnderTestN
             {
                 // put the references together in one string seperated by ", "
```

Comment:
Suggestion:
```csharp
                // put the references together in one string seperated by newlines
                string referencesString = string.Join(Environment.NewLine + "  ", projectReferences);
                string referenceChoise =
                    $"{Environment.NewLine}Choose one of the following references:{Environment.NewLine}  {referencesString} {Environment.NewLine}{Environment.NewLine}Example: --project-file={Path.GetFileName(projectReferences.First())}";

```

This adds a suggestion users could copy paste:
``` 
Test project contains more than one project references. Please add the --project-file=[projectname] argument to specify which project to mutate.
Choose one of the following references:
  ..\ExampleLibrary\ExampleLibrary.csproj
  ..\TestApp\TestApp.csproj

Example: --project-file=ExampleLibrary.csproj
```

Also I moved the Environment.NewLine to the string.Join
Also "whitch" to "which"